Grpcv2-PR8: Easy node functions (#130)
Concordium/concordium-node-sdk-js: packages/nodejs/READMEV2.md

Get next available sequence numbers for updating chain parameters after a given block.
```js
const blockHash = "fe88ff35454079c3df11d8ae13d5777babd61f28be58494efe51b6593e30716e";
const seqNums: NextUpdateSequenceNumbers = await client.getNextUpdateSequenceNumbers(accountAddress);
```

## shutdown
Shuts down the node.
```js
await this.client.shutdown();
```

## peerConnect

> Suggest to a peer to connect to the submitted peer details.

Reviewer: ```suggestion
Suggest to connect the specified address as a peer.
```

This, if successful, adds the peer to the list of given addresses, otherwise rejects.
Note. The peer might not be connected to instantly, in that case the node will try to establish the connection in near future.
```js
await this.client.peerConnect("127.0.0.1", 20000);
```

## peerDisconnect
Disconnect from the peer and remove them from the given addresses list
if they are on it. Resolves if the request was processed successfully.
Otherwise rejects.
```js
await this.client.peerDisonnect("127.0.0.1", 20000);
```

## getBannedPeers
Get a list of banned peers.
```js
const bannedPeers: IpAddressString[] = await this.client.bannedPeers();
```

## banPeer

> Bans a peer.

Reviewer: ```suggestion
Bans the specified peer.
```

Rejects if the action fails.
```js
await this.client.banPeer("127.0.0.1");
```

## unbanPeer

> unbans a peer.

Reviewer: ```suggestion
Unbans the specified peer.
```
